src/Routes/ImageManagerDetail/StatusLabel.js

```diff
@@ -16,12 +17,18 @@ const StatusLabel = ({ status }) => {
     icon = statusIcons[status];
     color = statusColors[status];
     text = imageStatusMapper[status];
-    text = text.charAt(0).toUpperCase() + text.slice(1);
+    text = text.charAt(0).toUpperCase() + text.slice(1).toLowerCase();
   }
   return (
-    <Label color={color} icon={icon}>
-      {text}
-    </Label>
+    <div className="label-container">
```

Comment:
My feeling is that `Label` is not needed. Try using [`Split` and `SplitItem`](https://www.patternfly.org/v4/layouts/split) and [spacing classnames](https://www.patternfly.org/v4/utilities/spacing) to set a space between the icon and text.
Maybe we can move the color in status color to the statusIcon using the color props.